backend/src/test/post.spec.ts

```diff
@@ -120,22 +144,33 @@ describe('mutations', () => {
                     data: { write: { title: 'Some post', id: expect.any(String) } }
                 });
         })
+
+        it('write a post without a token should fail', async () => {
+            server.requestOptions.context = () => ({});
+            await expect(action())
+                .resolves
+                .toMatchObject(
+                    {
+                        errors: [{message: 'Not Authorised!'}]
+                    }
+                );
+        })
     })
 
     describe('UPVOTE_POST', () => {
         const action = () => mutate({ mutation: UPVOTE_POST, variables: {id: getDummyPosts()[0].id,
                 voter: {name: getDummyUsers()[0].name}}});
         const UPVOTE_POST = gql`
-            mutation ($id: ID!, $voter: UserInput!) {
-                upvote(id: $id, voter: $voter) {
+            mutation ($id: ID!) {
+                upvote(id: $id) {
                     id
                     title
                     votes
                 }
             }
         `;
 
-        beforeEach(() => {
+        beforeEach(async () => {
```

Comment:
yayyyy async

Reply:
But here it's not necessary as there is no `await`.